week08_pomdp/practice_pytorch.ipynb

```diff
@@ -444,12 +466,50 @@
    "source": [
     "def to_one_hot(y, n_dims=None):\n",
     "    \"\"\" Take an integer tensor and convert it to 1-hot matrix. \"\"\"\n",
-    "    y_tensor = y.to(dtype=torch.int64).view(-1, 1)\n",
+    "    y_tensor = y.to(dtype=torch.int64).contiguous().view(-1, 1)\n",
```

Comment:
Are you sure this is required?

Reply:
Since it ensures the data in the memory is located correctly while changing the tensor's shape, why not? And since you are using `view` instead of `permute`, I think it should be added!